### PR TITLE
Update static-pod.md's default pod-manifest-path

### PR DESCRIPTION
**Pull request by nicklesimba**
kubeadm should be configured to use a pod-manifest-path of `/etc/kubernetes/manifests` by default, so this may be a better suggested directory to use than `/etc/kubelet.d/`

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
hi, i think having the consistency of the path between kubeadm docs and kubelet static pod docs is fine. so this change receives +1 from me.

but SIG Node is the owner of this page, so maybe they have other feedback.

/sig node cluster-lifecycle

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -90,10 +90,10 @@ For example, this is how to start a simple web server as a static Pod:
     EOF
     ```
 
-3. Configure your kubelet on the node to use this directory by running it with `--pod-manifest-path=/etc/kubelet.d/` argument. On Fedora edit `/etc/kubernetes/kubelet` to include this line:
+3. Configure your kubelet on the node to use this directory by running it with `--pod-manifest-path=/etc/kubernetes/manifests/` argument. On Fedora edit `/etc/kubernetes/kubelet` to include this line:
```
Comment (neolit123) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:93]: i'm not sure where the `/etc/kubernetes/kubelet` path is coming from and haven't seen it before in the k8s.io docs.

kubeadm uses different files for systemd / kubelet configuration:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:93]: I agree; even if it's wrong in the original, we should aim to fix this specific line.

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -67,12 +67,12 @@ For example, this is how to start a simple web server as a static Pod:
...
     ```
 
-2. Choose a directory, say `/etc/kubelet.d` and place a web server Pod definition there, for example `/etc/kubelet.d/static-web.yaml`:
+2. Choose a directory, say `/etc/kubernetes/manifests` and place a web server Pod definition there, for example `/etc/kubernetes/manifests/static-web.yaml`:
 
     ```shell
     # Run this command on the node where kubelet is running
-    mkdir /etc/kubelet.d/
-    cat <<EOF >/etc/kubelet.d/static-web.yaml
+    mkdir /etc/kubernetes/manifests/
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:74]: This won't work unless `/etc/kubernetes` already exists. Maybe use `mkdir -p`?

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (nicklesimba) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:93]: I am using a kind cluster and the `/etc/kubernetes/kubelet` path works for me on that environment.

**Comment by divya-mohan0209** [comment]
FYI, kind folks of @kubernetes/sig-node-pr-reviews : Please could you have a look at this PR and provide signoff from a technical standpoint?

**Comment by zacharysarah** [comment]
@nicklesimba Thanks for your patience. I see a +1 from @neolit123 and feedback addressed from @sftim, so I'm willing to approve this PR.

@kubernetes/sig-node-pr-reviews 👋🏻 Please feel free to address any feedback in a follow-on PR.

/lgtm
/approve

**Pull request merged**